netbox_topology_views/utils.py

```diff
@@ -177,7 +177,7 @@ def get_query_settings(request):
     if "show_neighbors" in request.GET:
         if request.GET["show_neighbors"] == "on" :
             show_neighbors = True
-    
+
```

Comment:
You did actually nothing here except housekeeping. Please do not touch anything that does not belong to the PR. Otherwise there is more to review than necessary.

Reply:
Fixed, Github cannot figure it out due to more changes in this block.